src/lib/functions/map/set_all_to_value.ts

```diff
@@ -1,4 +1,4 @@
-export const set_all_to_value = <K, V>(map: Map<K, V>, value: V) => {
+export const set_all_to_value = <K, V>(map: SvelteMap<K, V>, value: V) => {
```

Comment:
Why is this a `SvelteMap`?

Reply:
Map deprecated

Reply:
I think there's no need for a reactive map in this case. It's just a plain JS function :)

Reply:
![image](https://github.com/user-attachments/assets/67cf20b9-d71f-48bb-a010-1cb7a8964ebf)

Reply:
IDK, the suggested it.... U r right anyways